mysql-test/lib/My/SafeProcess/CMakeLists.txt

```diff
@@ -14,14 +14,17 @@
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
 
 
- IF (WIN32)
+IF (WIN32)
   ADD_EXECUTABLE(my_safe_process safe_process_win.cc)
   ADD_EXECUTABLE(my_safe_kill safe_kill_win.cc)
   TARGET_LINK_LIBRARIES(my_safe_kill dbghelp psapi)
 ELSE()
   ADD_EXECUTABLE(my_safe_process safe_process.cc)
 ENDIF()
 
+ADD_EXECUTABLE(wsrep_check_version wsrep_check_version.c)
```

Comment:
Make sure that this is build only when WITH_WSREP is defined i.e. not on WIN32 and other not supported environments.

Reply:
I added the necessary checks to the make files, plus checks in the code of the .pl / .pm modules, now the extra utility is not going to be built without WITH_WSREP